src/pages/Settings/SettingsPage.vue

```diff
@@ -182,7 +182,11 @@ export default {
 	},
 	apollo: {
 		query: pageQuery,
-		prefetch: true,
+		preFetch(config, client) {
```

Comment:
I don't think this is necessary and can be `preFetch: true,`

Reply:
When I do `preFetch: true` the content isn't there on the initial render. A few seconds later it jumps in causing a page reflow. Is there something else I should be doing to prevent that reflow?

Reply:
I stand corrected. Seems fine now. Not sure what I was seeing before.